dist/flatpak/com.github.input-leap.input-leaps.yml

```diff
@@ -10,34 +10,63 @@
 # https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1013
 
 app-id: com.github.inputleap.InputLeap
-runtime: org.gnome.Platform
-runtime-version: '42'
-sdk: org.gnome.Sdk
-command: input-leap-flatpak
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: input-leap
 
 finish-args:
     # We want to talk to the network
     - --share=network
     # Need to read our config from somewhere
     - --filesystem=xdg-config/input-leap
-    # HACK: so we can put our EIS socket somewhere accessible
-    # This won't be needed once the portal actually supports our interfaces
-    - --filesystem=xdg-run/eis-0
     # Let's show off :)
-    - --nosocket=x11
-    - --nosocket=wayland
+    - --socket=x11
+    - --socket=wayland
+    - --env=QT_QPA_PLATFORM=wayland
+    - --device=dri           # For hardware acceleration if needed
+    - --env=DISPLAY=:0       # Ensure the DISPLAY variable is set correctly
```

Comment:
is this really needed?

Reply:
which part do you think isn't needed?

Reply:
forcibly setting `DISPLAY=:0`. This should be correctly set by flatpak and in some cases the display may actually have a different value so setting it here would prevent it from connecting.

Reply:
I also can't test this as it doesn't work on my setups because input-leaps and input-leapc need to run with --use-ei and currently input-leap GUI doesn't start them with that flag. This bug prevents me from testing on wayland based systems. I could try on Xorg potentially.

Reply:
I can try removing this

Reply:
removed and removed QT_QPA_PLATFORM too.